### ISSUE TITLE
Tabview: Ensures elements with an ARIA role that require child roles contain them

### ISSUE THREAD

**Issue by Bryelmo** [primefaces/primeng#13802]
### Describe the feature you would like to see added

It would be better for Tabview component pass the accessibility WCAG 2.1 AA standard test. I'm using it in a project that requires this standard.

### Is your feature request related to a problem?

Tabview component fails a WCAG 2.1 AA standard test with Axe Devtools and Accessibility insights for web extensions on Google Chrome browser.
`<ul role="tablist" class="p-tabview-nav">
 `

### Describe the solution you'd like

Axe Devtools says how to solve this problem, you need to fix at least (1) of the following:

- Element has children which are not allowed: li[tabindex]
- Element uses aria-busy="true" while showing a loader

**Comment by mertsincan** [comment]
Hi,

So sorry for the delayed response! Improvements have been made to many components recently, both in terms of performance and enhancement. Therefore, this improvement may have been developed in another issue ticket without realizing it. You can check this in the documentation. If there is no improvement on this, can you open a new issue so we can include it in our roadmap?

Thanks a lot for your understanding!
Best Regards,
